workflow/metrics/scripts/metrics/gene_scores.py

```diff
@@ -0,0 +1,131 @@
+import anndata as ad
+import scanpy as sc
+import numpy as np
+import scib
+
+from utils.accessors import subset_hvg
+
+def get_anndata_for_gene_score(_adata, _adata_raw, _gene_signature, _var_key) -> float:
+    _ad = ad.AnnData(X=_adata_raw.X, var=_adata_raw.var, obs=_adata.obs, obsp=_adata.obsp, uns=_adata.uns)
+
+    # TODO: is this needed?
+    if 'feature_name' in _ad.var.columns:
+        _ad.var_names = _ad.var['feature_name']
```

Comment:
This is neccessary for datasets where the gene name is no in `.var_names`, but in `adata.var['feature_name']`, which is the case for datasets from CELLxGENE. However, I think it makes sense to move this code to the runner script. We already have it for the `adata`, but not `adata_raw`, where we are getting the genes from.

```suggestion
```

Could you  make the according change to the runner script and test that this commit works for your pipeline?